test/e2e/common/container_probe.go

```diff
@@ -45,7 +45,7 @@ const (
 	defaultObservationTimeout = time.Minute * 4
 )
 
-var _ = framework.KubeDescribe("Probing container", func() {
+var _ = framework.KubeDescribe("[sig-node] Probing container", func() {
```

Comment:
Don't we want to have SIGDescribe here to avoid adding "sig-node" everywhere, e.g.:

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/framework.go#L22

Reply:
SIGDescribe works as a common package-level function and here we have multiple sigs in a single package.
Unless you propose adding SIGNodeDescribe, SIGStorageDescribe etc.

Reply:
Yeah - that's what I was suggesting in the case of test/e2e/common

Reply:
FTR I opened https://github.com/kubernetes/kubernetes/pull/99016 to cleanup a bit the existing tests.

Reply:
https://github.com/kubernetes/kubernetes/pull/99016/files is dealing with this so marking this as resolved.